web-registry/src/pages/Projects/hooks/useProjectsSellOrders.utils.tsx

```diff
@@ -40,6 +43,11 @@ export const getPurchaseInfo = ({
     .map(order => parseFloat(order.quantity))
     .reduce((total, quantity) => total + quantity, 0);
 
+  const userCreditsAvailable = ordersForThisProject
```

Comment:
this is just a nit. but the name of this confused me for a minute. "user credits available" had my brain thinking about "these are the credits that user has available for sale". but after a few minutes reading the code, it did click. "user credits available" means "the credits available for a user to purchase". i'm not strongly opinionated on this but an alternative could be "creditsAvailableForUser". but naming is hard and subjective, and i'm not convinced my suggestion is better 😆, so not a blocker by any means

Reply:
makes sense!

Reply:
Will change it, thanks for the suggestion!